packages/core/src/App/Containers/RealAccountSignup/terms-of-use.jsx

```diff
@@ -68,6 +68,28 @@ const IOMDescription = () => (
     </React.Fragment>
 );
 
+const MaltaDescription = () => (
+    <React.Fragment>
+        <h4>
+            <Localize i18n_default_text={'Jurisdiction and choice of law'} />
+        </h4>
+        <p>
+            <Localize
+                i18n_default_text={
+                    'Your account will be opened with Binary ( Europe ) Ltd, and will be subjected to the jurisdiction and laws of Malta Gaming Authority.'
```

Comment:
Is there supposed to be spaces around `Europe` in the parentheses?

Reply:
The whole flow is under copy check, will update all the texts once it is done.